crates/optimism/rpc/src/eth/mod.rs

```diff
@@ -67,6 +74,9 @@ pub struct OpEthApi<N: RpcNodeCore> {
     /// Sequencer client, configured to forward submitted transactions to sequencer of given OP
     /// network.
     sequencer_client: Option<SequencerClient>,
+    /// List of addresses that _ONLY_ return storage proofs _WITHOUT_ an account proof when called
+    /// with `eth_getProof`.
+    storage_proof_only: Vec<Address>,
```

Comment:
we should wrap all of those fields in a new Arced `OpEthApiInner` type because this is cloned per request